src/main/java/study/racingcar/controller/RacingGameController.java

```diff
@@ -0,0 +1,20 @@
+package study.racingcar.controller;
+
+import study.racingcar.domain.*;
+import study.racingcar.view.InputView;
+import study.racingcar.view.ResultView;
+
+import java.util.List;
+
+public class RacingGameController {
+    public void start() {
+        List<String> carNames = InputView.getCarList();
+        int time = InputView.getTime();
+
+        MovableDistance movableDistance = new RandomMovableDistance();
+        RacingGame racingGame =
+                new RacingGame(carNames, movableDistance);
+        RacingGameResult racingGameResult = racingGame.play(new Time(time));
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        RacingGameResult racingGameResult = racingGame.play(time);
```